docs/operation/listening_processes.rst

```diff
@@ -0,0 +1,29 @@
+Listening processes
+===================
+
+In MetalK8s context several processes are deployed and they need to communicate
+with each other, sometimes locally, sometimes between machines in the cluster,
+or with the end user.
+
+Several addresses must be free for MetalK8s processes depending on the
+node roles.
+
+.. jinja:: salt_values
+
+   {%- for role, info in listening_processes.items() %}
+
+   {% if role == "node" %}
+     {%- set role = "all" %}
+   {%- endif %}
+   Listening processes on {{ role }} nodes
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   .. csv-table::
+      :header: "Address", "Description"
+      :align: left
```

Comment:
To make things neat, I'd suggest applying constant width to all tables. Maybe adding `:width: 100%` would be enough, in this case.